### PR TITLE
finish issue 19

### LINKED ISSUE #19: Templete
complete the templete for: login.html index.html owner.html register.html nothing.html
and slove tthe problem between bootstrap4 and django2.2 can not run together.
also need to write a views to makes every templete work

### PR DESCRIPTION
**Pull request by Yihao0118**
makd every page works, finish templete for: login;restaurant owner;creat account;index;

### CODE REVIEW

File: templates/footer.html
Comment (mashathepotato) [inline templates/footer.html:1]: We don't really need a link to food.com. Can just leave the copyright to look professional.

**Review by mashathepotato — COMMENTED** [review]
Not sure if this will conflict with Mohamed's work on registration and views

**Comment by mashathepotato** [comment]
@MSH1084 can you review this merge request? There are some minor conflicts with main, but I'm more concerned about there being overlap with your work.

File: templates/footer.html
Comment (Yihao0118) [reply templates/footer.html:1]: I'll do that

**Comment by Yihao0118** [comment]
> @MSH1084 can you review this merge request? There are some minor conflicts with main, but I'm more concerned about there being overlap with your work.

Or you can merge your work first and then I'll change my template and link it to your views

**Pull request merged**